docs/DeveloperGuide.md

```diff
@@ -680,11 +733,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 Given below are instructions to test the app manually.
 
-<div markdown="block" class="alert alert-info">
 
-**:information_source: Note:**<br>
+<div markdown="span" class="alert alert-info">
 
-These instructions only provide a starting point for testers to work on;
+**:information_source: Note:** These instructions only provide a starting point for testers to work on;
 testers are expected to do more *exploratory* testing.
```

Comment:
I tried fixing this in a previous PR, can you help to change this back to the markdown="block"

Reply:
Alright got it.